first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
I did that and checked the needles with that tag and they all appeared to be wrong. However I have checked again and found that my working directory is a bit out of date and that someone has since added new gdm needles which should match for my scenario: https://github.com/os-autoinst/os-autoinst-needles-opensuse/commit/18fa6eb19e7acca2cabebaaaa21ee568e91e3fb0